index.js

```diff
@@ -14,11 +14,10 @@
  * Module dependencies.
  */
 
-var os = require('os');
+var osName = require('os-name');
 
-var USER_AGENT = 'node/' + process.version
-  + ' (' + process.platform + ' ' + os.release()
-  + '; ' + process.arch + ')';
+var USER_AGENT = 'Node.js/' + process.version.slice(1)
```

Comment:
Hmm, that's pretty new. They're not even sure where yet to put it and I don't see any examples of what it's value would look like either. However, yes, supporting this would be a great idea when it's available.